Makefile

```diff
@@ -5,10 +5,22 @@ LDFLAGS := $(shell llvm-config --cxxflags --ldflags --libs core native --system-
 
 .PHONY: all clean
 
-all: test.bc
+all: test.bc fibonacci_rec.bc fibonacci_iter.bc multitime
```

Comment:
I'll do `make multitime` then and add the C files as dependencies for multitime.